config/modules/replicate-bagit.cfg

```diff
@@ -0,0 +1,61 @@
+#---------------------------------------------------------------#
+#-------BagIt-based AIP REPLICATION CONFIGURATIONS--------------#
+#---------------------------------------------------------------#
+# Configuration properties used solely by BagIt-based AIP       #
+# Replication Tasks. These settings are only utilized if your   #
+# 'replicate.cfg' file is configured to use the BagIt packaging #
+# format (packer.pkgtype = bagit) and you've enabled one or     #
+# more replication tasks in your curate.cfg file.               #
+#---------------------------------------------------------------#
+
+# The BagIt-based AIP replication tasks just wrap calls to the default
+# DSpace AIP Backup & Restore tool: 
+# https://wiki.duraspace.org/display/DSDOC18/AIP+Backup+and+Restore
+# The below settings allow you to custom how the AIP Backup & Restore
+# feature is called. This provides you additional control over
+# which options you want enabled/disabled during replication.
+
+#### BagIt Bag Profile ####
+
+# The Bag Profile setting allows you to select a BagProfile which the RTS
+# will create and read bags for. The RTS will check the conformance of a
+# bag to a profile as part of both the packaging and restoration processes.
+#
+# See: https://github.com/duraspace/bagit-support/ for more information
+#
+# Available Options: aptrust, beyondtherepository
+# Default: beyondtherepository
+
+replicate-bagit.profile = "beyondtherepository"
```

Comment:
I'm slightly surprised by the quotes around each value in this cfg file. If you look closely at other `.cfg` files, we never use quotes around values. They aren't even used when spaces exist in values, see for example: https://github.com/DSpace/dspace-replicate/blob/master/config/modules/curate.cfg#L63-L77

My preference would be to update this cfg file to no longer use/require quotes around the values.  This would require changing every configuration in this file however, as they all have quotes